### ISSUE TITLE
error while running npm i

### ISSUE THREAD

**Issue by godwnwaswa** [ethereumjs/ethereumjs-monorepo#2790]
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/data/data/com.termux/files/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:203:23)
gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:293:12)
gyp ERR! System Linux 4.19.127-g85ba30884861-dirty
gyp ERR! command "/data/data/com.termux/files/usr/bin/node" "/data/data/com.termux/files/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /data/data/com.termux/files/home/projects/ethereumjs-monorepo/node_modules/c-kzg
gyp ERR! node -v v19.9.0
gyp ERR! node-gyp -v v9.3.1
gyp ERR! not ok
npm ERR! code 1
npm ERR! path /data/data/com.termux/files/home/projects/ethereumjs-monorepo/node_modules/c-kzg
npm ERR! command failed
npm ERR! command sh -c node-gyp rebuild

**Comment by acolytec3** [comment]
If you run `npm install --omit=optional`, this should skip the `c-kzg` dependency and hopefully avoid this issue.  Just note that you won't be able to work with EIP 4844 transactions without `c-kzg` (if that's a concern).

**Comment by godwnwaswa** [comment]
thanks for the response, let me try again I'll provide feedback.

**Comment by godwnwaswa** [comment]
i tried but am still getting the same error. i want to spin up a local
private blockchain using all these packages, how do i go about it?

**Comment by acolytec3** [comment]
I'm not sure why your build is failing (though since you're running in termux there are any number of possible missing dependencies you have).  If you can get all the dependencies to install, you can then run the client in "dev" mode via something like:
`node dist/bin/cli.js --dev=poa` from `[repo root]/packages/client` and that will start the client running locally on an ephemeral PoA network and you can interact with it via the RPC using the standard Ethereum JSON-RPC.  That would be relatively similar to how ganache works today, though we don't prefund accounts with our code.  It can be done, but it's not part of the preconfigured options.

**Comment by godwnwaswa** [comment]
again thanks, am working on a decentralized e-commerce platform, my plan was to develop a private ethereum-standard blockchain to specifically handle all the smart contracts for this project. I have worked on a minimal nodejs blockchain but it's too experimental that's why am running around looking to form a not-so-complex ethereum implementation, any suggestions? I'll appreciate.

**Comment by acolytec3** [comment]
Hmm, that's a tricky one.  If you can get our client up and running in your environment, it's probably the only game in town in Javascript.  Ganache and Hardhat provide "local node" sort of features, though I don't know if either provides persistent state as part of their offerings.  If you're not intent on using Javascript, I'd say just run geth as your client.  It's easy enough to set up with a private network configuration.   

Feel free to join [our discord](https://discord.gg/px5BbJzX) and ask in the `#client` channel and we can triage further.

**Comment by godwnwaswa** [comment]
let me hop on the discord, thanks

**Comment by gabrocheleau** [comment]
Is this issue still active? Were you able to resolve it @godwnwaswa ?
